pkg/utils/k8sutil.go

```diff
@@ -372,6 +372,17 @@ func EnsureK8sResources(ns, name string, funcObj *spec.Function, client kubernet
 		addInitContainerAnnotation(dpm)
 	}
 
+	// update deployment for memory request
+	if funcObj.Spec.Memory != "" {
+		//mem value has already been validated before
+		quantity, _ := resource.ParseQuantity(funcObj.Spec.Memory)
```

Comment:
Yeah I agree to store the parsed quantity in the function Spec so it won't be re-parsed. I throw away the error here because there is already a validation at the client-side